### PR TITLE
Drag and drop support

### LINKED ISSUE #10: Implement drag & drop functionality

### PR DESCRIPTION
**Pull request by Mafii**
This implements #10 

Closes #10

### CODE REVIEW

**Comment by jeremystucki** [comment]
Can you add a screen recording to the PR?

**Review by jeremystucki — CHANGES_REQUESTED** [review]
Not working as intended

- Drag any module above another
- Both modules are now the same

**Comment by Mafii** [comment]
> Not working as intended
> 
>     * Drag any module above another
> 
>     * Both modules are now the same

Fixed now, can you check again?

File: src/components/Module.vue [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
-  </button>
-  <h3 class="has-text-weight-bold">{{module.name}}</h3>
-  <p>{{module.ects}} ECTS</p>
-</div>
+  <Draggable>
+    <div
+      class="column module mt-1"
+      :key="module.name"
+      v-bind:style="{
+        'background-color': this.$parent.$parent.$parent.getColorForCategory(module.categories[0])
```
Comment (jeremystucki) [inline src/components/Module.vue:7]: So beautiful

File: src/components/Module.vue [hunk 2]
```diff
@@ -26,7 +33,7 @@ export default {
   },
   methods: {
     removeModule(moduleId) {
-      this.$parent.$parent.removeModule(this.semesterNumber, moduleId);
+      this.$parent.$parent.$parent.removeModule(this.semesterNumber, moduleId);
```
Comment (jeremystucki) [inline src/components/Module.vue:36]: <3

**Review by jeremystucki — APPROVED** [review]

**Pull request merged**